src/blocks/buttons/edit.js

```diff
@@ -16,6 +16,8 @@ import Controls from './controls';
  */
 import { Component, Fragment } from '@wordpress/element';
 import { InnerBlocks } from '@wordpress/block-editor';
+import { createBlock, getBlockType } from '@wordpress/blocks';
+import { select, dispatch } from '@wordpress/data';
```

Comment:
I think this requires to declare a new `wp-data` dependency when [loading the editor assets](https://github.com/godaddy-wordpress/coblocks/blob/a75783f0c05b38d1939196dc628fe65f8cc6e04a/includes/class-coblocks-block-assets.php#L117), just in case Gutenberg doesn't load it by default in a future release.

Reply:
Thank you for the feedback. The dependency has been added.